### DISCUSSION TITLE
The Feed(s) I Want: PR Activity, Commits, or Both - Filtered by Organization or Repos I Choose

### DISCUSSION THREAD

**Discussion by danielBingham** [community/community#13446] [Other Feature Feedback, Questions, & Ideas]
The new algorithmic feed gives me absolutely nothing of value - it's giving me a bunch of information I have no need for or interest in.  It's just adding pure noise and no signal.  

The following feed might work for people who use the commit as the unit of review, but since there's no way to filter it to a certain subset of repos, it's also way too noisy to be useful.  And for those of us who use the Pull Request as the unit of review, it's absolutely useless.  Again, all noise and no signal.

I would love the ability to construct an arbitrary number of custom feeds to surface information I actually need.

I want to create new feeds, select whether they contain commits, pull request activity (PRs opened, closed, merged, and reviewed, and commented on), or both.  Then I want to be able to filter which repos/organizations that feed is limited to.  Ideally, I'd be able to select a set of feeds to show on the homepage as tabs I could click between to quickly scan activity in the subsets of repos I work on.

So for example, I would create one feed to be my "Primary Responsibility area at Work".  This would be the subset of repos I am primarily responsible for at my workplace.  I'd configure it to show PR activity - new PR opens, new reviews, new comments  (on review or on the PR), and PR merges or closes.  This would surface the information I need and would allow me to very quickly scan to see whether there were any PRs I needed to look over or PR discussions I wanted to add my voice to.  This is not a view I have right now.  Notifications doesn't surface this information in a way that's easy to scan, and otherwise I have to go digging through (potentially) multiple boards or issues lists to see it.  Instead being able to quickly scan down a feed of comments or reviews and reply right in the feed, I have to click through numerous screens and it's easy to miss stuff.  

I would then create a second feed that would be "All of work" which I would configure to be just PR activity for my work organization.  This would allow me to quickly scan activity in other repos that I'm not primarily responsible for, and contribute if there are things of value I could add.  Right now I can't do this at all, I have to rely on my teammates tagging me in.  It's just much too time consuming to try to scan through all the activity - even though, were it presented in feed format of just the new stuff, there's probably not so much that I couldn't easily scan it every day.  It's the amount of digging through screens and trying to figure out what's new that's time consuming.  

Then I'd create a feed for "My Stuff" which would contain all activity on repos I own (PRs and Commits).  This would be mostly empty since, for the most part, I'm not collaboration with folks right now.  But someday I'd like to, and this feed would be very useful then.

Finally I'd create various feeds for the all the open source I follow based on what project is, how interested in it I am, and how involved in I am.  

Having this ability - the ability to create multiple custom feeds to surface this information I want by groups that are meaningful to me - has the potential to make collaboration on Github much easier and more efficient.  One of the problems we face as a team is knowing when we need to respond to a pull request.  And right now we mostly solve that with process (pinging each other, standups) and tooling (Jira/Slack).  Having a feed we could quickly check for new activity would significantly grease those wheels and save us time elsewhere.

**Comment by clarle** [comment]
+1, my home algorithmic feed is not information I need most of the time. 

https://github.com/pulls/review-requested is a better feed for me today than anything else, and I default to going to this page as my GitHub "homepage" nowadays.

**Reply by tyhsq** [reply]
But even "Review Requested" is insufficient when one has multiple simultaneous comment threads on different PRs. It's hard to keep track of different conversations happening at the same time.

Having a "feed" for PR conversations fixes this. It's like the "unreads" feed in Slack, but in GItHub

**Comment by siminapasat** [comment]
We're in the planning / design phase of a very similar concept 👍 As things come together, we'll loop back to make sure it covers your scenarios

**Reply by danielBingham** [reply]
Great!  One thing to include, that I realized I didn't mention, would be issue activity as well.  I just discovered that someone commented on an issue of mine 28 days ago.  It's a comment I would have responded to immediately had I known it existed, but Notifications are too noisy and I simply didn't notice it.  A feed that bubbles that stuff to the top that I could set as my home page would be fantastic!
